docs/drilldowns/explore-underlying-data.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[explore-underlying-data]]
+== Explore data underlying visualizations
+
+Dashboard panels have "Explore underlying data" actions that will redirect
+you to *Discover* narrowing down documents to ones likely used in
+the visualization.
+
+"Explore underlying data" actions are available for visualizations which are
+backed by a single index pattern.
+
+There are two "Explore underlying data" actions: (1) one which appears in the
+panel's context menu; (2) and the other which appears when user clicks or brushes
+the chart.
+
+[float]
+[[explore-data-from-panel-context-menu]]
+=== Explore data from panel context menu
+
+Context menu "Explore underlying data" action will navigate you to *Discover*,
+to the index pattern used in the visualization and will carry over all filters,
+query, and time range applied to the visualization.
+
+[role="screenshot"]
+image::images/explore_data_context_menu.png[Explore underlying data from panel context menu]
+
+[float]
+[[explore-data-from-chart]]
+=== Explore data from chart action
+
+In-chart "Explore underlying data" action will also navigate you to *Discover*,
```

Comment:
I would re-phrase this a bit to:
> In-chart "Explore underlying data" action will also redirect you to *Discover*, and will carry over the index pattern, all filters, query, and time range applied to the visualization, but it will also apply the filters and time range created from the click or brush event that triggered this action.

Reply:
Initiating **Explore underlying data** from the chart also navigates to Discover,
carrying over the current context.  In addition, this action applies the filters and time range
created by the events that triggered the action.

Reply:
Suggest:

> Initiating **Explore underlying data** from the chart also navigates to Discover,
> carrying over the current context for the visualization.  In addition, this action applies the filters and time range
> created by the events that triggered the action.

Reply:
Done.